pennylane_lightning/core/src/utils/CMakeLists.txt

```
@@ -11,6 +11,18 @@ if(NOT WIN32)
     list(APPEND LQUBIT_UTILS_FILES UtilLinearAlg.cpp)
 endif()
 
+set(CUDA_UTILS_ADDED FALSE)
```

we can cache this to ensure that this variable will be cached between CMake runs.

```suggestion
set(CUDA_UTILS_ADDED FALSE CACHE BOOL "Add cuda_utils header files"))
```